ros_gz_sim/launch/ros_gz_spawn_model.launch

```diff
@@ -0,0 +1,46 @@
+<launch>
+  <arg name="bridge_name" />
+  <arg name="config_file" />
+  <arg name="container_name" default="ros_gz_container" />
+  <arg name="namespace" default="" />
+  <arg name="use_composition" default="True" />
+  <arg name="use_respawn" default="False" />
+  <arg name="log_level" default="info" />
+  <arg name="world" default="" />
+  <arg name="file" default="" />
+  <arg name="string" default="" />
```

Comment:
Nit: I'd rename this parameter to `xml_string` to be consistent with `ros_gz_spawn_model.launch.py`.

Reply:
I tried to be consistent with [these changes](https://github.com/gazebosim/ros_gz/pull/602#issuecomment-2336614296).

Reply:
Changed it to `sdf_string`.

Reply:
Changed it to `world_string`.